shimmingtoolbox/cli/mask.py

```diff
@@ -128,3 +130,153 @@ def threshold(fname_input, output, thr):
     nib.save(nii_img, output)
     click.echo(f"The filename for the output mask is: {os.path.abspath(output)}")
     return output
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="""Creates a mask around the spinal cord using the Spinal Cord Toolbox (SCT). The mask, which
+                   size can be specified, requires to identify the spinal cord centerline. The method of identification
+                   is specified by the flag '--centerline'. The output of this function is a NIfTI file containing the
+                   mask.""")
+@click.option('--input', 'fname_input', type=click.Path(), required=True,
```

Comment:
Is there are reason why all the options for sct have a double dash?

Reply:
We are moving towards having all double dash options. Here was the discussion #222